### DISCUSSION TITLE
Tutorial of Implementing NVFlare on windows clients

### DISCUSSION THREAD

**Discussion by avramit5674** [NVIDIA/NVFlare#2651] [Q&A]
### Python version (`python3 -V`)

3.8

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.3.8

### NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)

2.3

### Operating system

Windows

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [X] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

As described in this Q&A question - https://github.com/NVIDIA/NVFlare/discussions/2142, running NVFlare on Windows clients is based on a Docker environment. Could you please provide a comprehensive tutorial on how to set up a real-world example of NVFlare using Windows clients, or offer suggestions and best practices? Our products are installed on Windows machines, and we would greatly appreciate assistance with implementing NVFlare.

**Comment by YuanTingHsieh** [comment]
We don't have like a step by step tutorial, but I can share some pointers.

You can check https://docs.docker.com/desktop/install/windows-install/

And you can use the base image: nvcr.io/nvidia/pytorch:24.03-py3

There are some good materials about docker you can search for it.
Once you have docker installed and the image up and running, you can do pip install nvflare inside the docker environment and play with NVFlare.
